script/hypercube/hypercube.sh

```diff
@@ -585,7 +571,7 @@ function monitoring_yunohost() {
 
     date >> $tmplog
     echo -e "\n" >> $tmplog
-    yunohost tools diagnosis --debug &>> $tmplog
+    yunohost tools diagnosis &>> $tmplog
```

Comment:
N.B. : this command gonna get removed in 3.8 ... personally I never found this command so useful except to display yunohost version, which can also be obtained from yunohost --version :/

Reply:
Ok thanks for pointing this, I will remove it for preparing the v3.8

Reply:
If someone know the utility of `monitoring_yunohost()` because I think we can remove it ?